### PR TITLE
Include date filter on View Logs page

### PR DESCRIPTION
**Pull request by mariadanieldeepak**

### CODE REVIEW

File: include/Core/UI/ListTable/LogListTable.php [hunk 1]
```diff
@@ -312,4 +312,40 @@ public function prepare_items() {
 	public function no_items() {
 		_e( 'Your email log is empty', 'email-log' );
 	}
+
+	/**
+	 * Displays the search box.
+	 *
+	 * @since 2.0
+	 * @access public
```
Comment (sudar) [inline include/Core/UI/ListTable/LogListTable.php]: We don't need to specify access for `public` methods

File: include/Core/UI/ListTable/LogListTable.php [hunk 2]
```diff
@@ -312,4 +312,40 @@ public function prepare_items() {
...
+	 *
+	 * @param string $text     The 'submit' button label.
+	 * @param string $input_id ID attribute value for the search input field.
+	 */
+	public function search_box( $text, $input_id ) {
+		if ( empty( $_REQUEST['s'] ) && !$this->has_items() )
```
Comment (sudar) [inline include/Core/UI/ListTable/LogListTable.php]: WordPress coding standard excepts us to always use `{}`

File: include/Core/UI/ListTable/LogListTable.php [hunk 3]
```diff
@@ -312,4 +312,40 @@ public function prepare_items() {
...
+			return;
+
+		$input_text_id  = $input_id . '-search-input';
+		$input_date_id  = $input_id . '-search-date-input';
+		$input_date_val = ( ! empty( $_REQUEST['d'] ) ) ? $_REQUEST['d'] : '';
```
Comment (sudar) [inline include/Core/UI/ListTable/LogListTable.php]: We should always sanitize user input

File: include/Core/UI/ListTable/LogListTable.php [hunk 4]
```diff
@@ -312,4 +312,40 @@ public function prepare_items() {
...
+		if ( ! empty( $_REQUEST['order'] ) )
+			echo '<input type="hidden" name="order" value="' . esc_attr( $_REQUEST['order'] ) . '" />';
+		if ( ! empty( $_REQUEST['post_mime_type'] ) )
+			echo '<input type="hidden" name="post_mime_type" value="' . esc_attr( $_REQUEST['post_mime_type'] ) . '" />';
+		if ( ! empty( $_REQUEST['detached'] ) )
+			echo '<input type="hidden" name="detached" value="' . esc_attr( $_REQUEST['detached'] ) . '" />';
+		?>
+		<p class="search-box">
+			<label class="screen-reader-text" for="<?php echo esc_attr( $input_id ); ?>"><?php echo $text; ?>:</label>
+			<input type="search" id="<?php echo esc_attr( $input_date_id ); ?>" name="d" value="<?php echo $input_date_val; ?>" placeholder="Search by date" />
```
Comment (sudar) [inline include/Core/UI/ListTable/LogListTable.php]: Text not available for translation.

File: include/Core/UI/ListTable/LogListTable.php (hunk 1 above)
Comment (mariadanieldeepak) [inline include/Core/UI/ListTable/LogListTable.php]: Oops, somehow I missed this. Fixed it now.

File: include/Core/UI/ListTable/LogListTable.php (hunk 2 above)
Comment (mariadanieldeepak) [inline include/Core/UI/ListTable/LogListTable.php]: Oops, somehow I missed this. Fixed it now.

File: include/Core/UI/ListTable/LogListTable.php (hunk 3 above)
Comment (mariadanieldeepak) [inline include/Core/UI/ListTable/LogListTable.php]: Got it. JS validation was done. But missed this. Used `sanitize_text_field()`

File: include/Core/UI/ListTable/LogListTable.php (hunk 4 above)
Comment (mariadanieldeepak) [inline include/Core/UI/ListTable/LogListTable.php]: Fixed.

**Comment by mariadanieldeepak** [comment]
> Also I am curious. Where did you get the images from? I just want to make sure we have license to use them.

From jQuery UI. I just used the images that we require and ignored the reset.

**Comment by sudar** [comment]
@mariadanieldeepak I have replaced jQuery UI with bower and merged this pull request.

Just check it out once to make sure all your changes are in place.

**Pull request merged**